lib/matplotlib/patches.py

```diff
@@ -380,6 +380,8 @@ def set_edgecolor(self, color):
         Parameters
         ----------
         color : :mpltype:`color` or None
+            Note that any alpha value included in this color will be ignored
+            unless the patch's alpha is set to ``None``.
```

Comment:
I suggest not to add this note to all color parameters (of which we have plenty), but instead go for a centralized description as mentioned above (either in the target of ``:mpltyle:`color` `` or linked from there.